data/projects/bdk.mdx

```diff
@@ -1,7 +1,7 @@
 ---
 title: 'Bitcoin Dev Kit'
 dateAdded: '2022-12-29'
-summary: 'The goal of the Bitcoin Dev Kit (BDK) project (originally called Magical Bitcoin 🧙) is to provide libraries and tools that make it 10x easier to build secure, full featured, cross-platform on-chain Bitcoin wallets. All BDK components are designed to be lightweight and modular so that they can be adapted for virtually any use-case: from single-sig mobile wallets to multi-billion-dollar cold storage vaults.'
+summary: 'Seamlessly build cross-platform mobile wallets.'
```

Comment:
+Bitcoin to the summary.
'Seamlessly build cross-platform mobile Bitcoin wallets.'

Reply:
Just trying to learn on the fly here, so let me know if I'm wrong. But I think you could/should propose your edits as a "commit" directly on the code and then I can accept/decline your "commit" because I created the "PR". Alternatively, I could also just take your suggestion and create a new commit to the same PR. Am I getting this right?

Reply:
Hey @lucasmogu 
It might be a permission issue, in that I am not able to edit content for the descriptions directly, thereby leaving me with the option of commenting and requesting a change. The Commit tab doesn't seem to allow for this either. Check out my user experience:

https://github.com/OpenSats/website/assets/141419722/7714bd42-1dfe-4ae6-b9db-edc816c512b6

Reply:
Ok @Arvin21M, I think you're right, I do recall having the same issue in the past and I think @dergigi had to clear my access to making commits. I'll go ahead and add your suggestions now.

@dergigi when you have a chance, after the 50+ job interviews you're doing, could you please look into this access for @Arvin21M ?

Reply:
@Arvin21M the way to add changes to a PR is to suggest changes via comments. It's the little blue `+` that shows up on the left side of each line when you hover over the lines.

See these two links for more info:

1. https://haacked.com/archive/2019/06/03/suggested-changes/
2. https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request

The person in charge of the PR can then accept or reject your changes, similar to how you would [suggest edits in a Google Doc](https://www.howtogeek.com/398789/how-to-suggest-an-edit-in-google-docs/).

I hope this makes sense.

Reply:
Thanks @dergigi. 

That's the way I supplied the comments to @lucasmogu  for each specific line initially (using that + symbol). 

What Lucas and I were trying to do is see if I can make edits/commits to _his_ edits/commits to make a new PR instead of commenting and having Lucas implements my changes from my comments, but I think that would involve me pulling down a Branch from Lucas in my GitHub, working it, pushing it back up, etc etc. 

So it seems like the method of just commenting directly on his PR is the way to go if I am understanding this correctly.

Also, thank you for all those links: I am putting those into a personal HowTo folder for future reference.

Reply:
@Arvin21M I think you're missing one step to be able to turn your edits into commits that the PR owner can then approve or decline. This article shows it well https://haacked.com/archive/2019/06/03/suggested-changes/
![Screenshot 2023-10-13 091939](https://github.com/OpenSats/website/assets/81401034/a0184cbd-b4a1-4a91-88ae-52b06c44e639)

Reply:
Thanks @lucasmogu. You're right! I did miss that one step you circled. Looking forward to trying again on the next review.